src/styles/components/_specification-block.scss

```diff
@@ -18,32 +18,40 @@ $breakpoint-tablet: 820px;
 
   ul {
     list-style: url("../../../public/images/teal-decorative-bullet.svg");
+
+    @media (prefers-color-scheme: dark) {
+      list-style: url("../../../public/images/pink-decorative-bullet.svg");
+    }
   }
 
   code {
-    color: map.get($colors, "blue");
+    color: var(--code-color);
   }
 
   a {
     text-decoration: underline;
-    color: map.get($colors, "black");
-    background: linear-gradient(to left, rgb(0 0 0 / 0%) 50%, map.get($colors, accent-teal) 50%);
+    color: var(--foreground-color);
+    background: linear-gradient(to left, transparent 50%, var(--accent-color) 50%);
     background-size: 201% 100%;
     background-position: right bottom;
     transition: ease-in-out all 250ms;
+    outline: none;
+  }
 
-    @media screen and (prefers-reduced-motion: no-preference) {
-      &:hover,
-      &:focus-within {
-        background-position: left bottom;
-      }
+  // hover/focus matched the footer text-links
+  @media screen and (prefers-reduced-motion: no-preference) {
+    a:hover,
+    a:focus-within {
+      background-position: left bottom;
+      color: var(--font-color-dark);
     }
+  }
 
-    @media screen and (prefers-reduced-motion: reduce) {
-      &:hover,
-      &:focus-within {
-        background-color: map.get($colors, "accent-teal");
-      }
+  @media screen and (prefers-reduced-motion: reduce) {
+    a:hover,
+    a:focus-within {
+      background-color: var(--accent-color);
+      color: var(--font-color-dark);
```

Comment:
Having all the text change color at once while the gradient slides from left to right feels a little clunky. I tried a few things to see if I could get the text to change color from left to right at the same rate as the background gradient, but it's honestly pretty tough. 
If you have time, I'd pair/ask on the dev channel to see if there's a solution to this, or maybe consider just having the bg color fade in instead of slide for dark mode.

Reply:
Do you think I should just focus on the `no-preference` reduce motion, correct? Or perhaps double the `reduce-motion` also? I guess the `reduce-motion` doesn't have the slide, so that should be ok, changing both the background-color & font color at the same time.

Reply:
What if I matched the hovers to the `related components` links instead?

Here's the commit for that: https://github.com/sparkbox/accessible-components/pull/76/commits/c82d5f17709b0368e3a76299672d639c2b258ead

Reply:
I'm think I'm being overly nitpicky and the original version's fine! It's honestly more important for it to be clear which item's currently focused than to have the transition be flawless, and the highlight does a better job of that than the slightly larger text. If we really want, we can revisit the animation later when we're done with the MVP.

...I also replaced that animation in the related components section as part of the details page changes 😓 It was messing with focus containers in a way I wasn't a fan of.